public/template/lib/layout.pug

```diff
@@ -29,6 +29,8 @@ html(lang="en")
 
     if !config.noweb
       <!--[if lt IE 9]><script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
+    
+    link(href="/humans.txt", rel="author")
```

Comment:
Thanks for you contribution, but what is the point of this?

Reply:
This was for #1674

Reply:
But we need expose it and check if exist

Reply:
yeah, I'm adding a file. Need help with including all names you'd want to see :)